build.sh

```diff
@@ -264,19 +264,25 @@ if [ ! "$NO_TEST" = 'true' ]; then
     date
     time docker run -t --rm "$GAME_IMAGE" 'printenv && ls -al'
     date
-    time docker run -t --rm "$GAME_IMAGE" "$GAME_BIN -game $GAME +version +exit" | tee "$TEST_DIR/test"
+    if  [ "$APPID" = 730 ]; then
+        time docker run -t --rm "$GAME_IMAGE" "game/bin/linuxsteamrt64/cs2 -dedicated +status +quit"
+    else
+        time docker run -t --rm "$GAME_IMAGE" "$GAME_BIN -game $GAME +version +exit" | tee "$TEST_DIR/test"
+    fi
     date
 
-    # Verify game version of the game image matches the value of GAME_VERSION
-    echo 'Verifying game image game version'
-    GAME_IMAGE_VERSION_LINES=$( cat "$TEST_DIR/test" | grep -iE '\bexe\b|version' | sed 's/[^0-9]//g' )
-    if ! echo "$GAME_IMAGE_VERSION_LINES" | grep -E "^$GAME_VERSION" > /dev/null; then
-        echo "Game version does not match GAME_VERSION=$GAME_VERSION"
-        echo 'GAME_IMAGE_VERSION_LINES:'
-        echo "$GAME_IMAGE_VERSION_LINES"
-        exit 1
+    if  [ ! "$APPID" = 730 ]; then
+        # Verify game version of the game image matches the value of GAME_VERSION
+        echo 'Verifying game image game version'
+        GAME_IMAGE_VERSION_LINES=$( cat "$TEST_DIR/test" | grep -iE '\bexe\b|version' | sed 's/[^0-9]//g' )
+        if ! echo "$GAME_IMAGE_VERSION_LINES" | grep -E "^$GAME_VERSION" > /dev/null; then
+            echo "Game version does not match GAME_VERSION=$GAME_VERSION"
+            echo 'GAME_IMAGE_VERSION_LINES:'
+            echo "$GAME_IMAGE_VERSION_LINES"
+            exit 1
+        fi
+        rm -f "$TEST_DIR/test"
     fi
```

Comment:
is there really no way to test version in cs2? it is important

Reply:
I've tried finding out, there doesn't appear to be a way so far.

Actually it isn't exactly important, `+app_update * validate` should update the game just fine.

Reply:
support added